neqo-transport/src/connection/tests/close.rs

```diff
@@ -40,7 +40,11 @@ fn connection_close() {
 
     client.close(now, 42, "");
 
+    let stats_before = client.stats().frame_tx;
     let out = client.process(None, now);
+    let stats_after = client.stats().frame_tx;
+    assert!(stats_after.connection_close == stats_before.connection_close + 1);
```

Comment:
```suggestion
    assert_eq!(stats_after.connection_close. stats_before.connection_close + 1);
```

Nit: would print both values on failure, making debugging easier.